collections/copernicus-dem.yaml

```diff
@@ -138,9 +138,9 @@ Summaries:
       data_type: float32
       statistics:
         minimum:
-          - 0
+          - -413
```

Comment:
Where is this data from? I found multiple sources indicating that the value is even lower (420 / 430m below sea level).

Reply:
It's from the STAC metadata of the DEM on the S3 bucket that we can believe has the correct values. I double checked an the min./max. is from -413/8848.